docs/framework/interop/importing-a-type-library-as-an-assembly.md

```diff
@@ -38,7 +38,7 @@ COM type definitions usually reside in a type library. In contrast, CLS-complian
   
 -   Visual Studio  
   
-     Visual Studio automatically converts COM types in a type library to metadata in an assembly. For instructions, see [How to: Add References to Type Libraries](../../../docs/framework/interop/how-to-add-references-to-type-libraries.md) and [Walkthrough: Embedding Type Information from Microsoft Office Assemblies](https://msdn.microsoft.com/library/85b55e05-bc5e-4665-b6ae-e1ada9299fd3(v=vs.100)).  
+     Visual Studio automatically converts COM types in a type library to metadata in an assembly. For instructions, see [How to: Add References to Type Libraries](../../../docs/framework/interop/how-to-add-references-to-type-libraries.md) and [Walkthrough: Embedding Type Information from Microsoft Office Assemblies (C# and Visual Basic)](https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2010/ee317478(v=vs.100)).  
```

Comment:
this exists on docs split: https://docs.microsoft.com/en-us/dotnet/visual-basic/programming-guide/concepts/assemblies-gac/walkthrough-embedding-type-information-from-microsoft-office-assemblies-in-vs

https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/assemblies-gac/walkthrough-embedding-type-information-from-microsoft-office-assemblies

apply the same changes on other topics that link to 85b55e05-bc5e-4665-b6ae-e1ada9299fd3